spanner/api/Spanner.Samples/InsertDatatypesDataAsync.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_insert_datatypes_data]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+public class InsertDatatypesDataAsyncSample
+{
+    public class Venue
+    {
+        public int VenueId { get; set; }
+        public string VenueName { get; set; }
+        public byte[] VenueInfo { get; set; }
+        public int Capacity { get; set; }
+        public List<DateTime> AvailableDates { get; set; }
+        public DateTime LastContactDate { get; set; }
+        public bool OutdoorVenue { get; set; }
+        public float PopularityScore { get; set; }
+    }
+
+    public async Task InsertDatatypesDataAsync(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+
+        byte[] exampleBytes1 = Encoding.UTF8.GetBytes("Hello World 1");
+        byte[] exampleBytes2 = Encoding.UTF8.GetBytes("Hello World 2");
+        byte[] exampleBytes3 = Encoding.UTF8.GetBytes("Hello World 3");
+
+        var availableDates1 = new List<DateTime>();
+        availableDates1.InsertRange(0, new DateTime[] {
+                DateTime.Parse("2020-12-01"),
+                DateTime.Parse("2020-12-02"),
+                DateTime.Parse("2020-12-03")});
+
+        var availableDates2 = new List<DateTime>();
+        availableDates2.InsertRange(0, new DateTime[] {
+                DateTime.Parse("2020-11-01"),
+                DateTime.Parse("2020-11-05"),
+                DateTime.Parse("2020-11-15")});
+
+        var availableDates3 = new List<DateTime>();
+        availableDates3.InsertRange(0, new DateTime[] {
+                DateTime.Parse("2020-10-01"),
+                DateTime.Parse("2020-10-07")});
+        List<Venue> venues = new List<Venue> {
+                new Venue {
+                    VenueId = 4, VenueName = "Venue 4", VenueInfo = exampleBytes1,
```

Comment:
nit: having several of these in on line is not common, maybe one per line.

Reply:
Done!